### ISSUE TITLE
Adding a DB Prefix

### ISSUE THREAD

**Issue by JimMackin** [salesagility/SuiteCRM#553]
Many web apps (Joomla, wordpress, e.t.c.) allow adding a DB prefix to tables. This allows having multiple apps on the same database (for example, for those on hosting that only allows one database).

Some considerations for this would need to be made however, existing third party plugins, e.t.c. that use raw SQL would no longer work and likely many parts of SuiteCRM would need to change.

See here: https://suitecrm.com/forum/installation-upgrade-help/6641-install-when-only-one-db-allowed

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Adding a DB Prefix](https://trello.com/c/Ee0XhHD3/143-adding-a-db-prefix)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
